src/Violation/Renderer/RendererInterface.php

```diff
@@ -10,4 +11,9 @@
      * @param Violation[] $violations
      */
     public function renderViolations(array $violations);
+
+    /**
+     * @param Error[] $errors
+     */
+    public function renderParserErrors(array $errors);
```

Comment:
@xabbuh @datenbrille This PR adds a `renderParserErrors` method to the `RendererInterface`.
Any suggestions how to display parser errors in the html output format ?